lib/lighthouse.js

```diff
@@ -13,7 +13,7 @@ const {
   getSignInCredentials
 } = require('./common');
 
-const LIGHTHOUSE_OPTS = {
+const getDefaultLighthouseConfig = () => ({
```

Comment:
Why's this a function?

Reply:
Is it because you're mutating the value you get from it?

Reply:
Yes. I'd be happy to do a deep copy of the object but felt it might be too much overhead for the value it would bring.